content.md

```diff
@@ -160,3 +160,29 @@
 ### 5.5 Add photo
 ![screenshot](./images/5/photo.jpg)
 <div class="pagebreak"></div>
+
+## 6. How to clear the application cache
+
+### 6.1 Start from home screen and swipe down from top of screen to display this interface
+![screenshot](./images/6/1.jpg)
+<div class="pagebreak"></div>
+
+### 6.2 Swipe down once more to display this swipe-down menu
+![screenshot](./images/6/2.jpg)
+<div class="pagebreak"></div>
+
+### 6.3 Tap *Apps*
+![screenshot](./images/6/3.jpg)
+<div class="pagebreak"></div>
+
+### 6.4 Scroll to Kandu app and tap the Kandu app bar
```

Comment:
Mention the list and the item in the list as before, avoid repetition and stay consistent with highlights: 'Scroll to the *Kandu* item in the list and tap it'